.github/workflows/ci.yml

```diff
@@ -164,6 +164,7 @@ jobs:
         pwn phd -l 0x3d --color=always /etc/os-release
 
         pwn checksec /bin/bash
+        (ulimit -v 500000 && pwn checksec /bin/bash)
```

Comment:
Maybe do `ulimit -v unlimited` afterwards again to avoid weird problems later.

Reply:
The `()` spawns a subshell so the ulimit limitation affects only that subshell and doesn't spill over to the outside:

```
root@pwndbg:~# ulimit -v
unlimited
root@pwndbg:~# (ulimit -v 500000 && pwn checksec /bin/bash)
[!] Could not populate PLT: Cannot allocate 1GB memory to run Unicorn Engine
[*] '/bin/bash'
    Arch:       amd64-64-little
    RELRO:      Full RELRO
    Stack:      Canary found
    NX:         NX enabled
    PIE:        PIE enabled
    SHSTK:      Enabled
    IBT:        Enabled
root@pwndbg:~# ulimit -v
unlimited
```

(Also I think that once a limit is set u can only set it back only being root; idk if we are root here.)